bokehjs/src/coffee/models/glyphs/xy_glyph.ts

```diff
@@ -7,6 +7,8 @@ export interface XYGlyphData extends GlyphData {
   _x: Arrayable<number>
   _y: Arrayable<number>
 
+  scale: Arrayable<number>
```

Comment:
This isn't being used anywhere.

Reply:
it's passed to `_render`

Reply:
Sure, but that's `HexTileData.scale` and this `XYGlyph.scale`. `HexTile` doesn't extend `XYGlyph`.